components/bl-video-recorder-component/src/helpers.js

```diff
@@ -0,0 +1,44 @@
+export const prepareToRecord = videoRef => {
+  if (videoRef) {
+    videoRef.autoplay = true;
+    videoRef.muted = true;
+    videoRef.controls = false;
+  }
+};
+
+export const prepareToView = videoRef => {
+  if (videoRef) {
+    videoRef.autoplay = false;
+    videoRef.muted = false;
+    videoRef.controls = true;
+    videoRef.srcObject = null;
+    videoRef.captureStream = null;
+  }
+};
+
+export const download = (blob, fileName) => {
+  const link = document.createElement('a');
+  link.style.display = 'none';
+  link.href = URL.createObjectURL(blob);
+  link.download = fileName;
+
+  document.body.appendChild(link);
+  link.click();
+
+  setTimeout(() => {
+    URL.revokeObjectURL(link.href);
+    link.parentNode.removeChild(link);
+  }, 0);
```

Comment:
I think it isn't crossplatform? Is there another way to implement download?

Reply:
I didn't find anything better.